Packs/Core/Playbooks/Playbook-Large_Upload_Alert.yml

```diff
@@ -0,0 +1,3959 @@
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 5e9e0b1d-0914-4277-874e-c035a26cdcfd
+    type: start
+    task:
+      id: 5e9e0b1d-0914-4277-874e-c035a26cdcfd
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -130
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 42c1a166-d133-4890-8f1f-405efe248dde
+    type: title
+    task:
+      id: 42c1a166-d133-4890-8f1f-405efe248dde
+      version: -1
+      name: Check Previous Similar Incidents
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 0
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: be76e8b9-ae9a-4c30-88e0-c6e65c30f47d
+    type: regular
+    task:
+      id: be76e8b9-ae9a-4c30-88e0-c6e65c30f47d
+      version: -1
+      name: Find Similar Alerts
+      description: |-
+        Finds similar incidents by common incident keys, labels, custom fields or context keys.
+        It's highly recommended to use incident keys if possible (e.g., "type" for the same incident type).
+        For best performance, it's recommended to avoid using context keys if possible (for example, if the value also appears in a label key, use label).
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: FindSimilarIncidents
```

Comment:
R u sure that's the right script? should be `searchalertsv2`

Reply:
There is a known issue with the 'FindSimilarAlerts' command not appearing in the master, even though it appears in XSAIM. 'FindSimilarIncidents' is the same command but named differently.

Reply:
Forgot about that - you are right. However, it's not this one you wanted to use? 
`DBotFindSimilarAlerts`